stdlib/public/SwiftOnoneSupport/SwiftOnoneSupport.swift

```diff
@@ -67,6 +132,14 @@ internal enum _Prespecialize {
       // force specialization of append.
       a.append(a[0])
 
+      #if _runtime(_ObjC)
```

Comment:
Why is this specific to ObjC?

Reply:
Because we compile different code without ObjC, so the symbols in the stdlib core are different.